collector/cfappscollector.go

```diff
@@ -40,12 +39,12 @@ func (c *CfAppsCollector) GetAnnotations() []string {
 
 //Collect gets the cf app metadata by querying the cf app. Assumes that the authentication with cluster is already done.
 func (c *CfAppsCollector) Collect(inputPath string, outputPath string) error {
-	cli, err := cfclient.NewConfigFromCF()
+	homeDir, err := os.UserHomeDir()
 	if err != nil {
-		logrus.Errorf("Error while getting cf config : %s", err)
+		logrus.Errorf("Error while getting current user's home directory: %s", err)
 		return err
 	}
-	client, err := cfclient.NewClient(cli)
+	client, err := getCfClient(homeDir)
```

Comment:
The function need not take home dir, since it is anyway being got in a fixed manner.

Reply:
Done